Gruntfile.js

```diff
@@ -1,28 +1,53 @@
 'use strict';
 
 module.exports = function(grunt) {
+  var fs = require('fs');
+  var path = require('path');
+  var isWindows = /^win/.test(process.platform);
 
   require('load-grunt-tasks')(grunt);
+  grunt.loadNpmTasks('grunt-contrib-concat');
 
+  // config
   var config = {
     libName: 'css-layout',
     distFolder: 'dist',
     srcFolder: 'src',
     testFolder: 'src/__tests__',
-    cTestFiles: 'src/__tests__/Layout-test.c src/Layout.c src/Layout-test-utils.c',
-    cTestOutput: 'c_test',
     javaLibFolder: 'src/java/lib',
     javaSource: 'src/java/tests/com/facebook/csslayout/*.java',
     javaTestFiles: 'org.junit.runner.JUnitCore com.facebook.csslayout.LayoutEngineTest com.facebook.csslayout.LayoutCachingTest com.facebook.csslayout.CSSNodeTest'
   };
 
+  // Create the dist folder if it doesn't exist. It is deleted by the 'clean' task.
```

Comment:
I'd prefer to see the clean task updated to `dist/**` - this feels a bit like a side-effect.